back/app/models/custom_page_file.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: custom_page_files
+#
+#  id             :uuid             not null, primary key
+#  custom_page_id :uuid
+#  file           :string
+#  name           :string
+#  ordering       :integer
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#
+# Indexes
+#
+#  index_custom_page_files_on_custom_page_id  (custom_page_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (custom_page_id => custom_pages.id)
+#
+class CustomPageFile < ApplicationRecord
```

Comment:
If it's the same as `StaticPageFile`, I would inherit from it. Though it's not 100% correct, it would indicate at least the logic is the same.

Reply:
Maybe you also don't need `CustomPageFileUploader`.

Reply:
I think we agreed that since `StaticPageFile` looks identical to this (and other such policies), we should stick to the convention for now (of duplicating rather than inheriting from a base class), and leave this as it is now in this PR, and consider changing later if needed/desired.

Likewise, I am assuming we also leave the `CustomPageFileUploader` as it currently is in this PR.